code/style.css

```diff
@@ -1,4 +1,385 @@
+  border-radius: 10px;
+  align-self: flex-end;
+}
+.nav1:hover, .nav2:hover, .nav3:hover, .nav4:hover{
+text-decoration:underline;
+color: #25838A;
+}
+.logo{
+  width: 50px;
+  height: 50px;
+  border: 3px solid #25838A;
+  border-radius: 50%;
+  padding: 5px;
+  -webkit-animation:spin 4s linear ;
+    -moz-animation:spin 4s linear ;
+    animation:spin 4s linear ;
+}
+
+@-moz-keyframes spin { 100% { -moz-transform: rotate(360deg); } }
+@-webkit-keyframes spin { 100% { -webkit-transform: rotate(360deg); } }
+@keyframes spin { 100% { -webkit-transform: rotate(360deg); transform:rotate(360deg); } }
+.logo-big{
+  width: 50px;
+  height: 50px;
+  border: 3px solid #25838A;
+  border-radius: 50%;
+  padding: 5px;
+  -webkit-animation:spin 4s linear ;
+    -moz-animation:spin 4s linear ;
+    animation:spin 4s linear ;
+}
+@-moz-keyframes spin { 100% { -moz-transform: rotate(360deg); } }
+@-webkit-keyframes spin { 100% { -webkit-transform: rotate(360deg); } }
+@keyframes spin { 100% { -webkit-transform: rotate(360deg); transform:rotate(360deg); } }
+.big-news{
+  width: 100%;
+  /* border: 2px solid #25838A; */
+  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  align-self: center;
+  max-width: 500px;
+
+}
+
+.big-news-image{
+  width: 95%;
+  padding: 1rem;
+  border-radius: 20px;
+  object-fit: cover;
+overflow: hidden;
+
+}
+
+.news-card{
+  width: 90%;
+  max-width: 500px;
+  max-height: 300px;
+  border: 2px solid #F2F2F2;
+  /* box-shadow: 5px 5px 5px #25838A; */
+  border-radius: 10px;
+  margin: 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  align-self: center;
+  box-shadow:3px 3px 10px #383C4B;
+}
+.news-card:hover{
+  box-shadow: 5px 5px 5px #25838A;
+}
+
+.news-card-image{
+  width: 95%;
+  height: 200px !important;
+  padding: 1rem 1rem 1rem 1rem;
+  border-radius: 20px;
+  object-fit: cover;
+  overflow: hidden;
+}
+
+h1{
+  color: #383C4B;
+  font-weight:900;
+  font-size: 2rem;
+  align-self: center;
+}
+
+h2{
+  color:#383C4B;
+  font-weight: 400;
+  margin: 0.5rem;
+  font-size: 1.5rem;
+}
+h3{
+  margin: 1rem 1rem;
+  align-self: flex-start;
+  color:#383C4B;
+  font-weight: 400;
+  font-size: 1.5em;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  /* max-height: 2em; */
+  display: -webkit-box;
+  -webkit-line-clamp:2; /* number of lines to show */
+  -webkit-box-orient: vertical;
+}
+
+.small-text{
+  color: #383C4B;
+  font-weight: 200;
+  margin: 0.5rem 0rem 2rem 0rem;
+  text-align: center;
+}
+.more-text{
+  color: #383C4B;
+  font-weight: 200;
+  margin: 0.5rem 0rem 2rem 0rem;
+}
+hr{
+  border: 2px solid #84ACB6;
+  width: 70%;
+  border-radius: 50px;
+  align-self: center;
+  max-width: 500px;
+  animation: fadeIn 10s;
+  opacity: 1;
+}
+
+
+@keyframes fadeIn {
+  from {
+  	opacity: 0;
+  }
+  to {
+ 	opacity: 1;
+  }
+}
+
+
+/*mobile*/
+@media (min-width:0px) and (max-width:767px){
```

Comment:
clever to do a lot of styles and then not to display it in the mobile. will try that next time instead of just adding a lot of new stuff to the tablet/desktop versions.